utils/parsing.go

```diff
@@ -247,12 +266,12 @@ func ParseSourceAndCohortId(c *gin.Context) (int, int, error) {
 }
 
 // separates a conceptIdsAndCohortPairs into a conceptIds list and a cohortPairs list
-func GetConceptIdsAndCohortPairsAsSeparateLists(conceptIdsAndCohortPairs []interface{}) ([]int64, []CustomDichotomousVariableDef) {
-	conceptIds := []int64{}
+func GetConceptIdsAndCohortPairsAsSeparateLists(conceptIdsAndCohortPairs []interface{}) ([]CustomConceptVariableDef, []CustomDichotomousVariableDef) {
+	conceptIds := []CustomConceptVariableDef{}
 	cohortPairs := []CustomDichotomousVariableDef{}
 	for _, item := range conceptIdsAndCohortPairs {
 		switch convertedItem := item.(type) {
-		case int64:
+		case CustomConceptVariableDef:
 			conceptIds = append(conceptIds, convertedItem)
```

Comment:
I think it is better to rename `conceptIds` here, as it is not a list of ids anymore, but a list of CustomConceptVariableDef

Reply:
Updated